presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -7238,6 +7261,8 @@ private Session withTimestampPrecision(Session session, String precision)
     {
         return Session.builder(session)
                 .setCatalogSessionProperty(catalog, "timestamp_precision", precision)
+                // TODO: remove when implementing https://github.com/prestosql/presto/issues/5170
+                .setCatalogSessionProperty(catalog, "collect_column_statistics_on_write", "false")
```

Comment:
Is this relevant? or did you disable stats collection for timestamp != 3 for now?

Reply:
Shouldn't be needed.  I missed it when I was cleaning up my code.